vulnerabilities/unchecked-return-values.md

```diff
@@ -0,0 +1,58 @@
+### Unchecked Return Values
+
+The main idea behind this type of vulnerability is the failure to properly handle the return values of external function calls. This can have significant consequences, including fund loss and unexpected behavior in the contract's logic.
+
+In Solidity, developers can perform external calls using methods like:
+
+1. `.send()`
+2. `.call()`
+3. `.transfer()`
+
+`.transfer()` is commonly used to send ether to external accounts, however, the `.send()` function can also be used. For more versatile external calls, the `CALL` opcode can be directly employed in Solidity.
+
+Each of these methods has a different behavior when it comes to error handling. The `.call()` and `.send()` functions return a boolean indicating if the call succeeded or failed. Thus, these functions have a simple caveat: the transaction that executes these functions (`.call()` and `.send()`) WILL NOT revert if the external call fails. Instead, `.call()` and `.send()` will simply return the boolean value `false`.
+
+A common pitfall arises when the return value is not checked, as the developer expects a revert to occur when, in reality, the revert will not occur if not explicitly checked by the developer.
+
+For example, if a contract uses `.send()` without checking its return value, transaction execution will continue even if the call fails, resulting in unexpected behavior. Take the below contract for example:
+
+```solidity
+contract Lotto {
```

Comment:
I like to comment above these unsafe example contracts `// INSECURE` to be extra clear that this code shouldn't be used.